### ISSUE TITLE
Periodic sync support?

### ISSUE THREAD

**Issue by penberg** [tursodatabase/libsql-experimental-python#33]
@glommer suggested that we add an optional `sync_period` parameter to `libsql.connect()`. When defined, it configures a periodic sync in seconds. If not defined or set to `None`, only calls to `sync()` will synchronize the database.

For example, in the following, you'd have a periodic sync every 500 milliseconds:

```
import libsql_experimental as libsql
import os

sync_url = os.getenv("LIBSQL_URL")
auth_token = os.getenv("LIBSQL_AUTH_TOKEN")

conn = libsql.connect("local.db", sync_url=sync_url, auth_token=auth_token, sync_period=0.5)
```

Suggested by @glommer.

**Comment by ipmb** [comment]
Yes, this would be super helpful... If it didn't exist I'd need to create it for my use case.

I'd like to be able to deploy a traditional web app and scale it horizontally. With sticky sessions and read-your-write, I can make sure a user always sees their own writes, but I still need to sync periodically to capture any writes that might happen on a different node.
